ui/src/app/layout/topics/schemasection/schema-section.component.html

```diff
@@ -58,6 +58,10 @@
                                 [disabled]="!newSchemaText || !newSchemaText.length" appSpinnerWhile
                                 #p="appSpinnerWhile" (click)="p.while(publishNewSchema())"><i
                             class="fas fa-file-upload mr-3"></i>{{ 'Publish Schema' | translate }}</button>
+
+                        <button class="btn btn-danger mt-3 ml-3"
+                                appSpinnerWhile #p="appSpinnerWhile" (click)="exitSchemaMode()"><i
+                            class="fas fa-times mr-3"></i>{{ 'Exit' | translate }}</button>
```

Comment:
Perhaps better "Cancel" (we should already have that as translation, which is missing here anyway)